src/zino/tasks/reachabletask.py

```diff
@@ -21,8 +21,9 @@ async def run(self):
         """Checks if device is reachable. Schedules extra reachability checks if not."""
         if self._extra_job_is_running():
             return
-        result = await self._get_sysuptime()
-        if not result:
+        try:
+            await self._get_sysuptime()
+        except TimeoutError:
```

Comment:
A note for future reference (not to be fixed by this PR): Depending on the original Zino implementation, this function should probably record the device boot time to its state.  I've added a utility function to `DeviceState` for this in one of the PRs I'm currently working on, as the original link state checker did this.